README.md

```diff
@@ -146,6 +146,23 @@ The Thread-specific parameters are listed below.
 | `pskd`                              | Pre-Shared Key for the device. |
 | `thread-master-key`                 | A key used to derive security material for MAC and MLE protection. |
 
+### Wi-SUN configuration
+
+The Wi-SUN -specific parameters are listed below.
+
+| Field                               | Description                                                   |
+|-------------------------------------|---------------------------------------------------------------|
+| `regulatory-domain`                 | Defines regulatory domain, refer to [ws_management_api](https://github.com/ARMmbed/mbed-os/blob/master/features/nanostack/sal-stack-nanostack/nanostack/ws_management_api.h) for correct values for your region. |
+| `operating-class`                   | Defines operating class in the regulatory domain |
+| `operating-mode`                    | Defines the operating mode |
+| `uc-channel-function`               | Unicast channel function |
+| `bc-channel-function`               | Broadcast channel function |
+| `uc-fixed-channel`                  | Fixed channel for unicast |
+| `bc-fixed-channel`                  | Fixed channel for broadcast |
+| `uc-dwell-interval`                 | Unicast dwell interval. Range: 15-255 milliseconds |
+| `bc-interval`                       | Broadcast interval. Duration between broadcast dwell intervals. Range: 0-16777216 milliseconds |
+| `bc-dwell-interval`                 | Broadcast dwell interval. Range: 15-255 milliseconds |
+| `network-name`                      | Network name for Wi-SUN the network, MUST be same for all the devices in the network |
```

Comment:
as network name is the most important I think it should be quite high on list

should we mention more on the regulatory domain, class, mode that those are defined in the Wi-SUN why specification

operation mode/class should we say that these define the network bandwidth and is limited by the regulatory domain